core/include/optee_msg_supplicant.h

```diff
@@ -48,14 +48,8 @@
  */
 
 /*
- * SQLite file system access
- */
-#define OPTEE_MSG_RPC_CMD_SQL_FS	8
```

Comment:
The IDs are used by tee-supplicant and OP-TEE OS only, and you're not supposed to mix different versions of supplicant and OP-TEE OS (or build an older tee-supplicant with include files from a newer OP-TEE os), so I don't think it is really useful.